src/insurance/Accident.java

```diff
@@ -1,15 +1,18 @@
-public class Accidents {
+package insurance;
+
+public class Accident {
     private int accidentsID;
     private Driver driver;
     private int secondaryID;
-    private String dateOfAccident;
+    //TODO should we store the full date or just the year?
+    private int yearOfAccident;
     private Boolean responsible;
 
-    public Accidents(int accidentsID, Driver driver, int secondaryID, String dateOfAccident, Boolean responsible) {
+    public Accident(int accidentsID, Driver driver, int secondaryID, int yearOfAccident, Boolean responsible) {
```

Comment:
We need to store the entire date in the db so this property is for retrieval of that. We could have another private variable yearOfAccident with a getter that accesses the dateOfAccident property and just returns the year. Thoughts?